src/trip/components/tripsTable.jsx

```diff
@@ -23,17 +26,17 @@ const TripsTable = ({ trips, users, isBack }) => {
         }
     }, [])
 
-    const handleDeleteTrip = () => {
+    const handleDeleteTrip = async (id) => {
+        await deleteTrip(id)
         setDeletePopup(false)
-        /** perform deletion from db */
     }
 
     return (
         <Table textAlign="center" size="small" celled striped compact>
             <Table.Header>
                 <Table.Row>
                     <Table.HeaderCell colSpan='5'>
-                        {title} {login ? <TripsModal mode='add' floated='right' isBack primary/> : null }
+                        {title} {login ? <TripsModal mode='add' floated='right' isBack={isBack && false} primary/> : null }
```

Comment:
Not sure to understand the change on isBack here

Reply:
puting only isBack alone was always interpreted as true, "isBack && false" enforce the value of isBack to be false if it is not passed as a prop

Reply:
Right, I didn't get the other changes you made at first